Assets/ChiselNodes/Test.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Chisel.Core
+{
+    public class Test : MonoBehaviour
```

Comment:
This should, of course, not be part of a non draft PR. But you already know that